cassandra/gather-seeds.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#How many retries before giving up
+RETRIES=30
```

Comment:
We could make it configurable, but I don't think it adds much value (and we already have a lot of configuration options as is).

If the retries are used up, the container will exit under an error condition and a new container will be started by the rc. The RETRIES values just determines how many times we try before exiting and restarting again.